File: killbill/day2/functions/3.py
def funca(x, y):
    print("x =", x)
    print("y =", y)
    print("----")
    return x+2, y+3, x+y

a, b, c = funca(100, 200)

print("a =", a)
print("b =", b)
print("c =", c)

#error
a, b = funca(100, 200)

File: chair/day4/copy_con/3.cpp
#include<iostream>
using namespace std;

class Windows
{
public:
  int ia;
  Windows()
  {
    cout << "normal constructor" << endl;
  }
  Windows(int val)
  {
    ia = val;
    cout << "normal constructor with 1 argument" << endl;
  }
  Windows(const Windows &obja)
  {
    ia = obja.ia;
    cout << "copy constructor" << endl;
  }
  ~Windows()
  {
    cout << "normal destructor" << endl;
  }
};

Windows funca(Windows objx){
  cout << "in function" << endl;
  return objx;
}

int main()
{
  Windows objx {200};
  Windows objy;
  objy = funca(objx);
  return 0;
}
